ethereum/contracts/BasicInboundChannelV2.sol

```diff
@@ -0,0 +1,66 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
Should overwrite `BasicInboundChannel.sol` with this file, rather than naming it `..V2.sol`.